pkg/chunkenc/memchunk_test.go

```diff
@@ -527,7 +527,7 @@ func TestSerialization(t *testing.T) {
 							require.Equal(t, labels.FromStrings("foo", strconv.Itoa(i)), logproto.FromLabelAdaptersToLabels(e.StructuredMetadata))
 						} else {
 							require.Equal(t, labels.EmptyLabels().String(), it.Labels())
-							require.Nil(t, e.StructuredMetadata)
+							require.Equal(t, labels.EmptyLabels(), logproto.FromLabelAdaptersToLabels(e.StructuredMetadata))
```

Comment:
I'm working off that world. There's `IsEmpty`

Reply:
Addressed in d9cba1b.